docs/pages/reference/signature-algorithms.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: Signature Algorithms
+h1: Signature Algorithms Reference
+description: "Signature algorithms used in Teleport."
+---
+
+An important role of the Teleport Auth Service is issuing short-lived SSH and
+TLS certificates to users and hosts that allow all connections to be
+authenticated, authorized, and encrypted.
+This page describes the various cryptographic signature algorithms used to sign
+each kind of certificate issued by Teleport.
+
+Historically, every Teleport-issued certificate was signed by a 2048-bit RSA key.
+RSA is one of the oldest and most well-known asymmetric cryptography algorithms.
+While it is still considered secure (as long as your keys are large enough),
+modern alternatives based on Elliptic-Curve Cryptography offer much better
+performance with smaller keys and better security.
+
+## Signature algorithm suites
+
+Unilaterally changing the signature algorithms used in Teleport would be a
+breaking change for our users, as certain environments may expect or rely on RSA
+signatures.
```

Comment:
This feels a bit too historical or detailed for a docs page.

If I remove this bit completely and re-read the entire section I don't feel like any info is lost. Should we just remove it?

Reply:
yeah, it's gone 👍